tests/unit-tests/src/VuFindTest/OaiPmh/HarvesterTest.php

```diff
@@ -261,50 +261,35 @@ protected function getFakeResponseWithToken()
 XML;
     }
 
-    /**
-     * Test that granularity is autoloaded by default.
-     *
-     * @return void
-     */
-    public function testGranularityAutoload()
```

Comment:
Similar to the removals in HarvesterFactoryTest we should no longer test granularity detection as a direct property of a new Harvester object.